src/locale/locales/fr/messages.po

```diff
@@ -1745,11 +1663,11 @@ msgstr "Pseudo"
 
 #: src/components/RichText.tsx:188
 msgid "Hashtag: {tag}"
-msgstr ""
+msgstr "Hashtag: {tag}"
```

Comment:
There should always be a space before a colon in `fr-FR`, either a regular one (if you don't know how to input a non-breaking space one, or don't care to) or a non-breaking space one.

💡 Tip! If there are two parts for the typographic sign, there should a non-breaking space next to it in `fr-FR`. Works with `!`, `?`, `:` and `;`!

```suggestion
msgstr "Hashtag : {tag}"
```